packages/messages/src/browser/notifications-contribution.ts

```diff
@@ -0,0 +1,93 @@
+/********************************************************************************
+ * Copyright (C) 2019 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import {
+    FrontendApplicationContribution, StatusBar, FrontendApplication, StatusBarAlignment, KeybindingContribution, KeybindingRegistry, KeybindingContext, Keybinding
+} from '@theia/core/lib/browser';
+import { NotificationsCommands } from './notifications-commands';
+import { CommandContribution, CommandRegistry } from '@theia/core';
+import { NotificationManager } from './notifications-manager';
+import { NotificationCenter } from './notification-center';
+
+@injectable()
+export class NotificationsContribution implements FrontendApplicationContribution, CommandContribution, KeybindingContribution {
+
+    protected readonly id = 'theia-notification-center';
+
+    @inject(NotificationManager)
+    protected readonly manager: NotificationManager;
+
+    @inject(NotificationCenter)
+    protected readonly notificationsCenter: NotificationCenter;
+
+    @inject(StatusBar)
+    protected readonly statusBar: StatusBar;
+
+    onStart(_app: FrontendApplication): void {
+        this.createStatusBarItem();
+    }
+
+    protected createStatusBarItem() {
+        this.updateStatusBarItem();
+        this.manager.onUpdate(e => this.updateStatusBarItem(e.notifications.length));
+    }
+    protected updateStatusBarItem(count: number = 0) {
+        this.statusBar.setElement(this.id, {
+            text: this.getStatusBarItemText(count),
+            alignment: StatusBarAlignment.RIGHT,
+            priority: 10,
+            command: NotificationsCommands.TOGGLE.id
+        });
+    }
+    protected getStatusBarItemText(count: number) {
+        return `$(bell-o) ${count ? ` ${count}` : '' }`;
```

Comment:
I think we could use the octicon bell here and align further with VSCode
https://octicons.github.com/icon/bell/